test/Dialect/FIRRTL/infer-widths.mlir

```diff
@@ -715,17 +715,20 @@ firrtl.circuit "Foo" {
 
   // CHECK-LABEL: @MemScalar
   // CHECK-SAME: out %out: !firrtl.uint<7>
-  firrtl.module @MemScalar(out %out: !firrtl.uint) {
+  // CHECK-SAME: out %dbg: !firrtl.ref<vector<uint<7>, 8>>
+  firrtl.module @MemScalar(out %out: !firrtl.uint, out %dbg: !firrtl.ref<vector<uint, 8>>) {
     // CHECK: firrtl.mem
+    // CHECK-SAME: !firrtl.ref<vector<uint<7>, 8>>
     // CHECK-SAME: data flip: uint<7>
     // CHECK-SAME: data: uint<7>
     // CHECK-SAME: data: uint<7>
-    %m_p0, %m_p1, %m_p2 = firrtl.mem Undefined {
+    %m_dbg, %m_p0, %m_p1, %m_p2 = firrtl.mem Undefined {
       depth = 8 : i64,
       name = "m",
-      portNames = ["p0", "p1", "p2"],
+      portNames = ["dbg", "p0", "p1", "p2"],
       readLatency = 0 : i32,
       writeLatency = 1 : i32} :
+      !firrtl.ref<vector<uint, 8>>,
```

Comment:
I love this debug port :heart_eyes: This makes the whole Grand Central mem tapping so much cleaner! :tada: